ext/lib/radio/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+if (CONFIG_HAS_SILABS_RAIL)
+  zephyr_include_directories(silabs_rail/v2.3/platform/emlib/inc)
```

Comment:
1. Is there  reason, you take the emlib includes from an external directory and not the checked in files in ext/hal/silabs/gecko/emlib ?

2. The directory layout of the gecko sdk (including rails) is different. What i mean is, the emlib include is there:
`SimplicityStudio_v4/developer/sdks/gecko_sdk_suite/v2.3/platform/emlib/inc` and the rails common folder there:
`SimplicityStudio_v4/developer/sdks/gecko_sdk_suite/v2.3/platform/radio/rail_lib/common`. I think it would be good if just one symbolic link would be sufficient to use rails and use the directory hierarchy without modifications.

Reply:
Sure, we can do that.